DerivationPath Alias (#471)
celo-org/developer-tooling: .changeset/tasty-schools-reply.md

---
'@celo/celocli': minor
---

> Add derivation alias for celoLegacy in account:new

Author: ```suggestion
For `account:new` command add alias "celoLegacy" to the `--derivationPath` flag 

```
